src/FsCheck/Runner.fs

```diff
@@ -120,6 +122,42 @@ module Runner =
                     yield EndShrink result
               yield! test newSize resize rnd1 gen
         }
+        
+#if PCL // sorry windows phone no parallelism for you
+    let private parallelTest limit = test
+#else
+    let internal parallelTest limit initSize resize rnd0 gen =
+        (initSize, rnd0)
+        |> Seq.unfold (fun (initSize, rnd) ->
+            let rnd1, rnd2 = Random.split rnd
```

Comment:
Note here that really, Random.split is intended to start a new "fork" of random numbers ( a new parallel thread/task) while the `next` methods is intended to get the next seed in the current fork. I still need to update this in the sequential runner too on this fork, but I would expect for the sequential runner that `split` is not called where it is now, and we use `next` instead. For the parallel case ideally a mix of the two is needed. This is necessary to have good randomness properties.